pkg/scheduler/framework/plugins/interpodaffinity/scoring.go

```diff
@@ -160,9 +160,14 @@ func (pl *InterPodAffinity) PreScore(
 		}
 	}
 
+	podInfo := framework.NewPodInfo(pod)
+	if podInfo.ParseError != nil {
```

Comment:
nit: add comment:

`This should have already failed on PreScore, unless extension point is disabled`

Reply:
Assuming you mean `PreFilter`? (this is prescore 🙂)

And I agree it's unlikely someone disables the filter plugin and leaves the score plugin, but definitely possible which is why it doesn't hurt to handle the error in both.

Based on https://github.com/kubernetes/kubernetes/pull/93660#discussion_r466634075, going to use `Error` from PreScore and `UnschedulableAndUnresolvable` from PreFilter. That sounds good to me

Reply:
sg!